app/controllers/api/v1/statuses_controller.rb

```diff
@@ -61,7 +61,7 @@ def create
                                          status_params[:status],
                                          status_params[:in_reply_to_id].blank? ? nil : Status.find(status_params[:in_reply_to_id]),
                                          media_ids: status_params[:media_ids],
-                                         sensitive: status_params[:sensitive],
+                                         sensitive: status_params[:spoiler_text].blank? ? status_params[:sensitive] : true,
```

Comment:
mmm, the additional checks i guess cause this to apply retroactively as well as going forward though.

could be nice. if it doesn't do any harm let it go that way?